### DISCUSSION TITLE
Problem on "make autotiler" when setting up the docker

### DISCUSSION THREAD

**Discussion by GBdangerous** [bitcraze/discussions#803] [Q&A]
When i am setting up docker and the autotiler and executing "make autotiler" , the connection went wrong, it just retry again and again. I am working on Ubuntu system, so how to solve it?
 
![图片](https://github.com/bitcraze/discussions/assets/46839372/4dd4abac-1da0-4bde-8668-bd25dbdac9c7)

**Comment by knmcguire** [comment]
Seems that there is an issue with the greenwaves server... you can perhaps wait a bit and try again, and if it still doesn't work, try to contact Greenwaves about this: https://greenwaves-technologies.com/
